dsl/camel-jbang/CamelJbang.java

```diff
@@ -0,0 +1,255 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//DEPS info.picocli:picocli:4.5.0
+//DEPS org.apache.camel:camel-core:3.12.0-SNAPSHOT
+//DEPS org.apache.camel:camel-core-model:3.12.0-SNAPSHOT
+//DEPS org.apache.camel:camel-api:3.12.0-SNAPSHOT
+//DEPS org.apache.camel:camel-main:3.12.0-SNAPSHOT
+//DEPS org.apache.camel:camel-kamelet-main:3.12.0-SNAPSHOT
+//DEPS org.apache.camel:camel-file-watch:3.12.0-SNAPSHOT
+//DEPS org.apache.camel:camel-resourceresolver-github:3.12.0-SNAPSHOT
```

Comment:
Yeah you can use BOM for the camel deps.
https://github.com/tadayosi/jbang-sandbox/blob/main/camel/camel.java#L2-L5

Reply:
Thanks @tadayosi! I will try that one today :)